tsconfig.json

```
@@ -0,0 +1,26 @@
+{
+    "compilerOptions": {
+      "target": "es5",
+      "lib": [
+        "dom",
+        "dom.iterable",
+        "esnext"
+      ],
+      "allowJs": true,
+      "skipLibCheck": true,
+      "esModuleInterop": true,
+      "allowSyntheticDefaultImports": true,
+      "strict": true,
+      "forceConsistentCasingInFileNames": true,
+      "noFallthroughCasesInSwitch": true,
+      "module": "esnext",
+      "moduleResolution": "node",
+      "resolveJsonModule": true,
+      "isolatedModules": true,
+      "noEmit": true,
+      "jsx": "react-jsx"
+    },
```

```suggestion
      "jsx": "react-jsx",
      "baseUrl": "src",
    },
```
위와 같이 baseUrl 속성을 지정해주면, 컴포넌트나 메소드를 임포트할 때 src 이후의 절대경로로 불러올 수 있어요! 예를 들어, 
```
import bigIcon from "../../assets/images/BigIcon.svg";
```
이렇게 임포트한 부분을
```
import bigIcon from "assets/images/BigIcon.svg";
```
이런식으로 바꿀 수 있어요!

기능상 문제는 없지만 만약, 디렉토리의 뎁스가 깊어지면 상대경로로 참조했을 때 가독성이 다소 떨어질 수 있어서,
절대경로로 임포트해봐도 좋을 것 같아요~~

---

오 이런 방법이..!! 앞으로는 이런식으로 해봐야겠네요~! 감사합니다 😊